ui/angular.json

```diff
@@ -138,19 +136,6 @@
             "codeCoverage": true
           }
         },
-        "lint": {
```

Comment:
Please keep the TS Linter if possible. See if the modification on this PR solves your problem here:
https://github.com/wgu-opensource/osmt/pull/352/files#diff-a2da1f8403af269843426c4ff06ba0d054bd9751d8adb2f39a9e53a321d9099fR139-R140

Reply:
# TSLint is deprecated
This is a message with Angular CLI 12
`TSLint's support is discontinued and we're deprecating its support in Angular CLI.`

> TSLint was removed on Angular 13

<br>

# Trying to keep lint property in angular.json

I can add again removed lines but when I run `ng lint` I see the next error

```
An unhandled exception occurred: Cannot find builder "@angular-devkit/build-angular:tslint".
See "/private/var/folders/m3/szvr8s3174n1jv9cz7k1v2w80000gn/T/ng-0we2MI/angular-errors.log" for further details.
```


# Conclusion
So it is not possible to keep TSLint; we can merge these changes without a linter and work to add ESLint in other ticket, or we can also migrate to ESLint in this PR.